src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -2075,6 +2075,29 @@ private static bool IsEquivalentSymbol(ISymbol declaredSymbol, ISymbol otherSymb
             foreach (IOperation operationBlock in operationBlocks)
             {
                 operationsToAnalyze.AddRange(operationBlock.DescendantsAndSelf());
+
+                // Special handling for IMethodBodyOperation and IConstructorBodyOperation.
+                // These are newly added root operation nodes for C# method and constructor bodies.
+                // However, to avoid a breaking change for existing operation block analyzers,
```

Comment:
NOTE: We have [existing unit tests](http://source.roslyn.io/#q=TestOperationBlockAnalyzer_EmptyMethodBody) to ensure we do not accidentally make this breaking change.